lib/puppet_x/binford2k/itemize/cli.rb

```diff
@@ -44,6 +44,11 @@ def to_s
         output << sprintf("    %#{name_width}s | %#{count_width}s\n", name, count)
       end
       output << "\n"
+
+      instances = counts.reduce(0) { |acc, c| acc + c[1] }
+
+      output << sprintf("    %#{name_width}s | %#{count_width}s\n", "Totals: #{counts.count}", instances)
```

Comment:
I don't love this precise format but can't think of better names at the moment. Got any ideas?